client/ayon_core/hosts/nuke/plugins/publish/collect_headless_farm.py

```diff
@@ -0,0 +1,39 @@
+import pyblish.api
+
+
+class CollectHeadlessFarm(pyblish.api.ContextPlugin):
```

Comment:
Does the headless mean that it will be published on farm from nuke? Because for that was chosen "Remote publishing" (run publishing on farm from a DCC).

Reply:
`headless` means publishing without the Publisher UI.

Reply:
So why is `"farm"` set to `True`?

Reply:
`farm` will make the publishing process skip integration.